jdi-light-html-tests/src/main/java/io/github/com/custom/ContactFormCustomFill.java

```diff
@@ -13,8 +13,7 @@
 import java.lang.reflect.Field;
 
 public class ContactFormCustomFill extends Form<Contacts> {
-    TextField name;
-    TextField lastName, position, passportNumber, passportSeria;
+    TextField name,  lastName, position, passportNumber, passportSeria;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4997679)